org.eclim.groovy/test/junit/org/eclim/plugin/groovy/command/complete/CodeCompleteCommandTest.java

```diff
@@ -58,4 +63,80 @@ public void completionByPrefix()
     result = results.get(1);
     assertEquals(result.get("completion"), "addAll(");
   }
+  
+  @Test
+  public void javaDoc(){
+    assertTrue("Groovy project doesn't exist.",
+        Eclim.projectExists(Groovy.TEST_PROJECT));
+    int count = getJavaDocCount("true");
+    // We do not compare with the actual number to make
+    // the test more robust ==> if something in eclipse
+    // changes and some elements do not exist anymore /
+    // do not have a javaDoc element the test still
+    // succeeds.
+    int minimalCount = 50;
+    Assert.assertTrue("We received some JavaDoc links", count > minimalCount);
```

Comment:
Please stick with the existing coding style and omit the `Assert.` prefix. Ditto for the other cases below.